src/lib/openapi/spec/public-signup-token-schema.ts

```diff
@@ -29,6 +29,7 @@ export const publicSignupTokenSchema = {
             description:
                 'The public signup link for the token. Users who follow this link will be taken to a signup page where they can create an Unleash user.',
             type: 'string',
+            nullable: true,
```

Comment:
I don't know what it means for this to be `null`, but it definitely can be in our tests. Does that mean we need to change our tests or is it actually nullable?